### DISCUSSION TITLE
异步物化视图刷新方式

### DISCUSSION THREAD

**Discussion by zhaoyangjun0328** [apache/doris#32452] [RFC]
大佬们，最新的doris支持异步物化视图，但是数据刷新只支持手动和定时两种方式，对于olap表有计划支持自动的近实时刷新吗？
或者说这种近实时的自动刷新有什么困难点，不好实现？

**Comment by maruyue** [comment]
中文问题，建议可以去ask.selectdb.com搜索一下，或者在哪里提问。这里可以尽量使用英文进行提问，方便来自其他国家的用户也可以查阅。
